### ISSUE TITLE
Daken for Ninjas

### ISSUE THREAD

**Issue by Vaneticorum** [DarkstarProject/darkstar#3834]
**_I have:_**

- [ ] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [ ] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:**
30170304_1

**_Server Version_** (type `@revision` in game) **:**
Unknown

**_Source Branch_** (master/stable) **:**
Master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**
Ninja 25: Daken not an available trait
https://www.bg-wiki.com/bg/Daken

**Comment by helixhamin** [comment]
Nope, it is not programmed, so will not be an available trait unless someone creates it.
